src/SprykerEco/Yves/Adyen/Plugin/Payment/KlarnaInvoicePaymentMapperPlugin.php

```diff
@@ -18,6 +18,13 @@
 class KlarnaInvoicePaymentMapperPlugin implements AdyenPaymentMapperPluginInterface
 {
     /**
+     * {@inheritDoc}
+     *
+     * Specification:
```

Comment:
* The word `Specification` is not needed in this case.
* The specification should go right after `{@inheritDoc}`.
*` \Spryker\Service\ProductConfigurationStorage\Plugin\PriceProduct\ProductConfigurationPriceProductVolumeFilterPlugin`